addons/mail/static/tests/translation/translation.test.js

```diff
@@ -31,22 +31,20 @@ test("Toggle display of original/translated version of chatter message", async (
     await start();
     await openFormView("res.partner", partnerId);
     await click("button[title='Expand']");
-    await contains("span[title='Translate']");
+    await contains("span[aria-label='Translate']");
```

Comment:
Could we remove the `spans`? If the label is pretty unique, it means we don't have to update all the test when the element is changed.